yarn.lock

```diff
@@ -1353,6 +1353,16 @@ ajv@^6.12.4:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+ajv@^8.12.0:
+  version "8.12.0"
```

Comment:
Do we need all these changes in yarn.lock @Chaitanya-019

Reply:
Not very sure